src/components/Todo/common/Todo/Form/index.tsx

```diff
@@ -59,12 +59,30 @@ export const Form = (props: AddProps) => {
               uuid: uuid(),
               isComplete: false,
             });
-            setItemName("");
+            setItemName('');
           }}
-          placeholder="Add item."
+          placeholder='Add item.'
           value={itemName}
-          className="w-10/12"
+          className='w-10/12'
           autoFocus
+          onKeyDown={(e) => {
```

Comment:
Please do it with react state instead of dom manupulation. 
You should declear a new state for focused item and use that to focus the item and manupulate that state on pressing up and down arrow.